board/freescale/mx6ullevk/mx6ullevk_siklu_pcb19x_se_environment.c

```diff
@@ -54,25 +54,48 @@ static sf_env_siklu_se_t* p_sf_env_siklu_se = &sf_env_siklu_se;
 
 static int syseeprom_access_init = 0;
 
-static const char siklu_default_environment_se[] = { //
-		"SE_product_name=8010FX" ";" //
-						"SE_mac=00:24:a4:00:de:ad" ";"//
-						"SE_board_serial=F123456789" ";"//
-						"SE_system_serial=0" ";"//
-						"SE_port_map=cd--" ";" "\0" //
-		};
+static const char siklu_default_8020_environment_se[] = {
```

Comment:
```suggestion
static const char siklu_default_8020F_environment_se[] = {
```

Reply:
Note: for the 8020FX, we need to define another list of variables. I'm not sure if we will be able to distinguish between 8020F and 8020FX using `siklu_get_board_type()`. For now, let's keep the implementation you did (with minor changes I suggested). Later, we will decide what to do with 8020FX.

Reply:
Ok